.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: go
+
+go:
+  - 1.10.x
+  - master
+
+env:
+  - DEP_VERSION="0.4.1"
+
+before_install:
+  # Install dep as per https://golang.github.io/dep/docs/FAQ.html#how-do-i-use-dep-in-ci
+  - curl -L -s https://github.com/golang/dep/releases/download/v${DEP_VERSION}/dep-linux-amd64 -o $GOPATH/bin/dep
```

Comment:
I know this is copy/pasted, but IMO wget is better or you should use `-f` with curl. Otherwise the command succeeds even for 403/404/etc.

Reply:
Good thinking, I'll add `-f`.

Reply:
Learnt the hard way 😉